tests/transformer/test_operators.py

```diff
@@ -0,0 +1,356 @@
+from RestrictedPython._compat import IS_PY35_OR_GREATER
+from tests import c_eval
+
+import pytest
+
+
+# Arithmetic Operators
+
+
+@pytest.mark.parametrize(*c_eval)
+def test_Add(c_eval):
+    result = c_eval('1 + 1')
+    assert result.code is not None
+    assert result.errors == ()
+    assert result.warnings == []
+    assert result.used_names == {}
+
+    eval_result = eval(result.code)
+    assert eval_result == 2
+
+
+@pytest.mark.parametrize(*c_eval)
+def test_Sub(c_eval):
+    result = c_eval('2 - 1')
+    assert result.code is not None
+    assert result.errors == ()
+    assert result.warnings == []
+    assert result.used_names == {}
+
+    eval_result = eval(result.code)
+    assert eval_result == 1
+
+
+@pytest.mark.parametrize(*c_eval)
+def test_Mult(c_eval):
+    result = c_eval('2 * 2')
+    assert result.code is not None
+    assert result.errors == ()
+    assert result.warnings == []
+    assert result.used_names == {}
+
+    eval_result = eval(result.code)
+    assert eval_result == 4
+
+
+@pytest.mark.parametrize(*c_eval)
+def test_Div(c_eval):
+    result = c_eval('10 / 2')
+    assert result.code is not None
+    assert result.errors == ()
+    assert result.warnings == []
+    assert result.used_names == {}
+
+    eval_result = eval(result.code)
+    assert eval_result == 5
+
+
+@pytest.mark.parametrize(*c_eval)
+def test_Mod(c_eval):
+    result = c_eval('10 % 3')
+    assert result.code is not None
+    assert result.errors == ()
+    assert result.warnings == []
+    assert result.used_names == {}
+
+    eval_result = eval(result.code)
+    assert eval_result == 1
+
+
+@pytest.mark.parametrize(*c_eval)
+def test_Pow(c_eval):
+    result = c_eval('2 ** 8')
+    assert result.code is not None
+    assert result.errors == ()
+    assert result.warnings == []
+    assert result.used_names == {}
+
+    eval_result = eval(result.code)
+    assert eval_result == 256
+
+
+@pytest.mark.parametrize(*c_eval)
+def test_FloorDiv(c_eval):
+    result = c_eval('8 // 2')
```

Comment:
I do not think that this is not a good example as the result does not differ from `8 / 2`, something like `7 // 2` would be better.